package.json

```diff
@@ -42,7 +42,9 @@
   },
   "devDependencies": {
     "@toznysecure/sdk": "^1.0.0-rc.4",
+    "@types/isomorphic-fetch": "^0.0.35",
     "@types/jest": "^26.0.24",
+    "@types/uuid": "^8.3.1",
```

Comment:
when converting code that makes use of external deps, their type definitions sometimes come from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) instead of the package itself.